src/pages/ResearcherProfile.js

```diff
@@ -108,6 +149,51 @@ export default function ResearcherProfile(props) {
     return updatedUser;
   };
 
+  const sendActions = async () => {
```

Comment:
i kind of split it out into two diff methods, lemme know if this works better!

I changed the variable name from actions -> supportRequests... I think i made it actions in the first place because I wasn't sure if they were all supposed to be support ticket requests, or if they would trigger other actions in the app!

Reply:
thank you!!